### PR TITLE
Add ESLint configuration

### PR DESCRIPTION
**Pull request by dcalder**
Add ESlint configuration:

- Add `.eslintrc` file.
- Add ESLint mention on Read Me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -833,6 +833,43 @@ Based on:
...
+
+    ```
+    {
+      "env": {
+        "browser": true,
+        "jquery": true
+      },
+
+      "rules": {
+        "block-scoped-var": [0],
```
Comment (atma) [inline README.md]: Why `block-scoped-var` is disabled? It helps to prevent situations when a variable is never defined due to some condition.

Comment (dcalder) [inline README.md]: Yes, I think the same. It's in that way to be consistent with the other linter configurations. See `.jshintrc`.

File: README.md [hunk 2]
```diff
@@ -833,6 +833,43 @@ Based on:
...
+        "guard-for-in": [1],
+        "indent": [2],
+        "no-caller": [2],
+        "new-cap": [2],
+        "no-eq-null": [0],
+        "no-eval": [2],
+        "no-extend-native": [2],
+        "no-loop-func": [0],
+        "no-plusplus": [2],
+        "no-shadow": [0],
```
Comment (atma) [inline README.md]: no-shadow is can be handy and help to prevent many error, why is it disabled?

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "env": {
+    "browser": true,
+    "jquery": true
+  },
+
```
Comment (battaglr) [inline .eslintrc]: Personally, I would remove this blank-line.

File: README.md [hunk 4]
```diff
@@ -833,6 +833,43 @@ Based on:
...
 
+- Use `eslint` with the following basic configuration:
+
+    ```
+    {
+      "env": {
+        "browser": true,
+        "jquery": true
+      },
+
```
Comment (battaglr) [inline README.md]: By adding `json` after the "```" we get syntax highlighting as in the other examples.

**Comment by battaglr** [comment]
Personally I would remove the full config from the readme —and the others as well— and just reference the files to avoid having to modify things twice in case of any change.

Are you OK with this changes, guys (@hmammana, @llinare, @atma)?

File: README.md (hunk 2 above)
Comment (dcalder) [inline README.md]: Same here.

File: README.md [hunk 5]
```diff
@@ -771,22 +771,22 @@ Based on:
...
         window.MyComponent = MyComponent;
-        
+
     }(this));
     ```
 
 ## Linting
 
-- Use [JSLint](http://www.jslint.com) or [JSHint](http://www.jshint.com) to:
+- Use [JSLint](http://www.jslint.com), [JSHint](http://www.jshint.com) o [ESlint](http://eslint.org/) to:
```
Comment (battaglr) [inline README.md]: Typo: it's "or" not "o". :wink:

File: README.md (hunk 1 above)
Comment (atma) [inline README.md]: Thanks, resolved.

File: README.md (hunk 5 above)
Comment (dcalder) [inline README.md]: Fixed.

**Pull request merged**